contributors.md

```diff
@@ -26,55 +26,53 @@ The following people are not part of the project team, but have been contributin
 * (qcz) - Scenery window, misc.
 * Matthias Lanzinger (lnz) - Climate, finance, scenario, ride reachability
 * (zsilencer) - Audio, multiplayer, misc.
-* Adrian Wielgosik (adrian17) - Misc.
-* (hexdec) - Misc.
-* Dennis Devriendt (ddevrien) - Misc.
-* Maciek Baron (MaciekBaron) - Misc.
+* (DutchRPW) - peep_update_days_in_queue, misc. money-related functions
+* Adrian Wielgosik (adrian17) - Ride window, top toolbar, map window, misc.
+* (hexdec) - Music credits window, staff window, misc.
+* Dennis Devriendt (ddevrien) - Banner window, map window, options window
+* Maciek Baron (MaciekBaron) - Peep enums, item flags, misc.
 * (AngeloG) - Scrollbar input, misc.
 * (jcdavis) - Misc.
 * (marcotc) - Rain drawing, misc.
-* (vanderkleij) - Misc.
-* Ben Pye (benpye) - Misc.
-* (JeroenSack) - Misc.
-* Sijmen Schoon (SijmenSchoon) - Misc.
-* Joe Minor Jr (wolfreak99) - Misc.
-* Inseok Lee (dlunch) - Original command line
-* Lewis Fox (LRFLEW) - Misc.
-* Marijn van der Werf (marijnvdwerf) - Drawing, misc.
-* Hielke Morsink (Broxzier) - Tile inspector, heightmap loader, misc.
+* (vanderkleij) - create_sprite, move_sprite_to_list
+* Ben Pye (benpye) - Logo rendering on title screen
+* (JeroenSack) - widget_scroll_get_part, misc.
+* Sijmen Schoon (SijmenSchoon) - redraw_peep_and_rain, misc bugfixes.
+* Lewis Fox (LRFLEW) - sub_69A997, OpenGL support, macOS
+* Marijn van der Werf (marijnvdwerf) - Peep functions, drawing and paint code
 
 ## Additional implementation (OpenRCT2)
 * (atmaxinger) - User configuration
 * (anyc) - Housecleaning, cross-platform fixes
 * Michael Steenbeek (Gymnasiast) - Cheats, RCT1 ride style, misc.
-* Miso Zmiric (mzmiric5) - Misc.
-* (DutchRPW) - Housecleaning, initialisation
-* Jørn Lomax (jvlomax) - User configuration
-* (KingHual) - Housecleaning
-* Alexander Overvoorde (Overv) - Misc.
-* (eezstreet) - Misc.
-* Thomas den Hollander (ThomasdenH) - Misc.
-* James Robertson (rd3k) - Misc.
+* Miso Zmiric (mzmiric5) - Twitch integration, misc.
+* Inseok Lee (dlunch) - Load save files from command line
+* Jørn Lomax (jvlomax) - Configuration parser
+* Alexander Overvoorde (Overv) - OpenGL improvements, Steam overlay detection, various bugfixes.
+* (eezstreet) - Add finances button to toolbar
+* Hielke Morsink (Broxzier) - Tile inspector, heightmap loader, misc.
+* Joe Minor Jr (wolfreak99) - Various cheats, bugfixes, new About and Changelog windows.
+* Thomas den Hollander (ThomasdenH) - Dithering in sprite importer, invert viewport dragging, park rating cheats misc.
+* James Robertson (rd3k) - Initial tile inspector, misc changes.
 * Robert Jordan (trigger-death) - UI theming, title sequence editor, misc.
 * Aaron van Geffen (AaronVanGeffen) - scenario select screen, font detection, misc.
 * Michał Janiszewski (janisozaur) - Linux port, crash handling, security, misc.
 * Kelson Blakewood (spacek531) - title sequences for release versions 0.0.4, 0.0.5/0.0.6/0.0.7, 0.1.0/0.1.1, and 0.1.2, title sequences using milliseconds
```

Comment:
How about `title sequences for release versions 0.0.4 through 0.1.2, title sequences using milliseconds`?